ScoutSuite/providers/aws/facade/iam.py

```diff
@@ -138,6 +138,21 @@ class IAMFacade(AWSBaseFacade):
 
         return roles
 
+    async def get_role_with_managed_policies(self, role_name):
```

Comment:
Roles' managed policies are already being fetched? Looks like this is designed specifically to get Lambda role policies.

Reply:
As this is specific for lambda, I will move it to the aws lambda facade